README.md

```diff
@@ -2,5 +2,247 @@
 
 [![Travis Build Status][build-badge]][build]
 
+This is a library of some custom validators for React components properties.
+Initially they were part of [React-Bootstrap](https://github.com/react-bootstrap/react-bootstrap/) project.
+
+### Usage
+
+All validators can be imported as
+```js
+import elementType from 'react-prop-types/elementType';
+// or
+import { elementType } from 'react-prop-types';
+...
+propTypes: {
+  someProp: elementType
+```
+or
+```js
+import CustomPropTypes from 'react-prop-types';
+// and then used as usual
+propTypes: {
+  someProp: CustomPropTypes.elementType
+```
+
+If you use `webpack`, you can bundle only those validators you need like this:
```

Comment:
If you use `webpack` and only want to bundle the validators you need, prefer the following approach: